src/hooks/useAnalytics.tsx

```diff
@@ -9,13 +10,12 @@ type AnalyticMessage = {
 };
 
 export default function useAnalytics() {
+  const analyticsEnabled = useAppSelector(getAnalyticsEnabled);
```

Comment:
Couldn't put this selector inside the callback bellow, as hooks can only sit inside functional components or other custom hooks. Is there some other non-hacky way to do this?

Reply:
How you've done it is correct!